src/Radicle/Internal.hs

```diff
@@ -13,3 +16,8 @@ import           Radicle.Internal.Parse as X
 import           Radicle.Internal.Pretty as X
 import           Radicle.Internal.PrimFns as X
 import           Radicle.Internal.Type as X
+
+createBindings :: (MonadIO m, ReplM m) => [Text] -> IO (Bindings (PrimFns m))
```

Comment:
Maybe we should call this `createReplBindings` or `createImpureBindings`. Please also add documentation.